website/docs/d/windows_web_app.html.markdown

```diff
@@ -0,0 +1,513 @@
+
+* `client_affinity_enabled` - Is Client Affinity enabled?
+
+* `client_cert_enabled` - Are Client Certificates enabled?
+
+* `client_cert_mode` - The Client Certificate mode.
+
+* `connection_string` - A `connection_string` block as defined below.
+
+* `custom_domain_verification_id` - The ID of the TODO.
+
+* `default_hostname` - The Default Hostname of the Windows Web App.
+
+* `enabled` - Is the Windows Web App enabled?
+
+* `https_only` - Does the Windows Web App require HTTPS connections.
+
+* `identity` - A `identity` block as defined below.
+
+* `kind` - The string representation of the Windows Web App Kind.
+
+* `location` - The Azure Region where the Windows Web App exists.
+
+* `logs` - A `logs` block as defined below.
+
+* `outbound_ip_address_list` - The list of Outbound IP Addresses for this Windows Web App.
+
+* `outbound_ip_addresses` - A string representation of the list of Outbound IP Addresses for this Windows Web App.
+
+* `possible_outbound_ip_address_list` - The list of Possible Outbound IP Addresses that could be used by this Windows Web App.
+
+* `possible_outbound_ip_addresses` - The string representation of the list of Possible Outbound IP Addresses that could be used by this Windows Web App.
+
+* `service_plan_id` - The ID of the Service Plan in which this Windows Web App resides.
+
+* `site_config` - A `site_config` block as defined below.
+
+* `site_credential` - A `site_credential` block as defined below.
+
+* `storage_account` - A `storage_account` block as defined below.
+
+* `tags` - A mapping of tags assigned to the Windows Web App.
+
+---
+
+A `action` block exports the following:
+
+* `action_type` - The predefined action to be taken to an Auto Heal trigger.
+
+* `custom_action` - A `custom_action` block as defined below.
+
+* `minimum_process_execution_time` - The minimum amount of time in `hh:mm:ss` the Windows Web App must have been running before the defined action will be run in the event of a trigger.
+
+---
+
+An `active_directory` block exports the following:
+
+* `allowed_audiences` - An `allowed_audiences` block as defined below.
+
+* `client_id` -  The ID of the Client used to authenticate with Azure Active Directory.
+
+* `client_secret` - The Client Secret for the Client ID.
+
+* `client_secret_setting_name` - The App Setting name that contains the client secret of the Client.
+
+---
+
+An `application_logs` block exports the following:
+
+* `azure_blob_storage` - An `azure_blob_storage` block as defined below.
+
+* `file_system_level` - The logging level.
+
+---
+
+An `application_stack` block exports the following:
+
+* `current_stack` - The Current Stack value of the Windows Web App.
+
+* `docker_container_name` - The name of the Docker Container in used.
+
+* `docker_container_registry` - The Container Registry where the Docker Container is pulled from.
+
+* `docker_container_tag` - The Docker Container Tag of the Container in use.
+
+* `dotnet_framework_version` - The version of .Net in use.
```

Comment:
Is this .net framework, .net (formally Core) or both?

Reply:
Both, the service handles this internally and refers to the setting as one thing. As discussed offline, I'll change this to `dotnet_version` to reduce the potential confusion.